contracts/core/ManualVolOracle.sol

```diff
@@ -0,0 +1,280 @@
+//SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.7.3;
+
+import {
+    IUniswapV3Pool
+} from "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {DSMath} from "../libraries/DSMath.sol";
+import {OracleLibrary} from "../libraries/OracleLibrary.sol";
+import {Welford} from "../libraries/Welford.sol";
+import {IERC20Detailed} from "../interfaces/IERC20Detailed.sol";
+import {Math} from "../libraries/Math.sol";
+import {PRBMathSD59x18} from "../libraries/PRBMathSD59x18.sol";
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract VolOracle is DSMath, AccessControl {
+    using SafeMath for uint256;
+
+    /// @dev The identifier of the role which maintains other roles.
+    bytes32 public constant ADMIN_ROLE = keccak256("ADMIN");
+
+    /**
```

Comment:
should we just gut everything else

Reply:
Yes, sounds good, only annualizedVol and its setter is needed

Reply:
Let's just leave `vol` in but return a 0